### PR TITLE
Pr update psmodels

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Comment by xresende** [comment]
@pnallin , there are other changes that need propagation to beaglebone hostnames and zabbix reconfiguration:
- There is now only one BBB for the TS dipole, `(LA-RaPS02:PS-UDC-TS1, LA-RaPS02:PS-UDC-TS2) -> LA-RaPS02:PS-UDC-TS`

File: beaglebone/beaglebone-udc.txt [hunk 1]
```diff
@@ -46,16 +46,19 @@ LA-RaCtrl:CO-PSCtrl-TB2     LA-RaPS06:PS-UDC-TB4    LA-RaPS06:PS-UDC-TB5    LA-R
 # ______ TS ______
 
 # --- TS Dipole ---
-LA-RaPS02:CO-PSCtrl-TS      LA-RaPS02:PS-UDC-TS1    LA-RaPS02:PS-UDC-TS2
+LA-RaPS02:CO-PSCtrl-TS      LA-RaPS02:PS-UDC-TS
```
Comment (fernandohds564) [inline beaglebone/beaglebone-udc.txt]: @xresende, I think there's a `1` missing at the end of the `UDC` definition:

`LA-RaPS02:PS-UDC-TS1`

File: beaglebone/udc-bsmp.txt [hunk 2]
```diff
@@ -102,12 +102,11 @@ IA-20RaPS02:PS-UDC-AS       IA-20RaCtrl:PS-DCLink-AS2 2
 # ______ TS ______
 
 # --- TS Dipole ---
-LA-RaPS02:PS-UDC-TS1        TS-Fam:PS-B 1
-LA-RaPS02:PS-UDC-TS2        TS-Fam:PS-B-Slave 2
+LA-RaPS02:PS-UDC-TS         TS-Fam:PS-B 1
```
Comment (fernandohds564) [inline beaglebone/udc-bsmp.txt]: @xresende, I think there's a 1 missing at the end of the UDC definition:

LA-RaPS02:PS-UDC-TS1

File: beaglebone/beaglebone-udc.txt (hunk 1 above)
Comment (xresende) [reply beaglebone/beaglebone-udc.txt]: done. thanks!

File: beaglebone/udc-bsmp.txt (hunk 2 above)
Comment (xresende) [reply beaglebone/udc-bsmp.txt]: done.thanks!

**Pull request merged**